src/MockBot.Api/Models/Chat.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.ObjectModel;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace MockBot.Api.Models
+{
+    // No logic so no unit tests are required
+    [ExcludeFromCodeCoverage]
+    public class Chat
+    {
+        public Guid Id { get; } = Guid.NewGuid();
+
+        [Required]
+        public Collection<Message> Messages { get; } = new Collection<Message>();
```

Comment:
<img width="1090" alt="image" src="https://user-images.githubusercontent.com/104758060/170096157-4c3312fe-60d4-4ce8-8134-a8a601477cd5.png">
Rider insists that it's a red-underline worthy error to make it a List and refuses to build otherwise. Not sure if that's a Rider specific thing if it isn't in our style guide.

Reply:
oh lol, sorry, I have little idea why sorry because I have not really used rider before, but maybe because the Messages object so far in the code only uses limited functionality at the moment such as Add or Remove

Reply:
So far it's even just Add :D

Would it be fine to leave as is and it can be changed to a List when necessary or do you have a suggestion of something I could do to it to emphasize its List-ness?